code/css-grid-assignment/index.css

```diff
@@ -0,0 +1,53 @@
+* {
+    box-sizing: border-box;
+}
+body {
+   margin: 0 auto;
+   padding: 10px 30px;
+   font-family: Arial, sans-serif;
+
+}
+#wrapper {
+    font-size: 2em;
+    gap: 1rem;
+    max-width: 1200px;
```

Comment:
This line wasn't necessary, it's okay for the page to stretch the full width. But it's okay!